services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/UserManagerImpl.java

```diff
@@ -247,12 +245,9 @@ public void deletePrincipal(UserInfo adminUserInfo, Long principalId) throws Not
 	public PrincipalAlias lookupUserForAuthentication(String alias) {
```

Comment:
change to lookupPrincipalForAlias. No logic in it. Callers must handle whether or not they get a Team